spec/resources/shadow-dom/control-image

```diff
@@ -1 +1 @@
jbjpnOFvym3hKbi+SSjy3V0+JIuLfynF6FlYfcXwa+UJZyYGoWjb+u76zPNPQyiSBUOUlY1G2Jo+vxSKiENEAqXFMCWWOAeOsz82rfO+tdslFBFeZMos13OXUHSUNd4kFM2z0Hkv+/Eov9fM/RChyHXbzvJN/Lb54S6hiO/sWvdNQpH9wC9kBxKK9NQjMNZKoARKoARKoARKoARKoARKoARKoARKoASuEQJTKCIO6J2yDpzOPgsa3H/hFibrgJpgtr43s8TR8msyk5Q5usno5eOz0xKK9LQgZAioez47aY+iOXYNwGdfEcFVfVfuPMpPLbNpzkooInjIviG2eHN+9gLZxdrY9wlFAufEkdlTyXf0UxIs9Nl8q/x+I4NIBoIguXJV+nn4QzgjLG0yGVezP5C36QlqxEeBaT1wZC5MwXB+/yT+Q4gidM4eQrKFZKdYs7XJAlEqbPbhsRcIEjJZHnEOQpFnb2OwHqugrVJcSgxOI2bJaiES7vP9db8UAWiN7e0XZn/bhwLzbCkU7duz+3xwi2u88Mf7hCJjIcDMEoFLoWjpX7KtlFTjW8usSA+SLcUnZAeyXULRPh+SdWFf2I9EIf7Gb/SMsi9nhtqu9d239qchFM2ziCAm+3CTUETkxcrZL/tz9iHD6Kj7Y9M671o71+8SipRz45PWU8lLtksoOsoabxKKZqk7LxxsGts+Pz9UKNp2lq/57fLDXUKR++iTt23dNwlFSmMq+6hPEeFc9tmmsoz79nI/L4ESKIESKIESKIESKIESKIESKIESKIESuKQEplDkjXyBY9lDa3vWCNQpsbbJ1sE+95RxQyAgpnhD3c8Ep12rTNRTRtBW8NYb7wLVM0C3fsahGUUEFGWnlGjTO0QAXSaJfi+zH9C69JxMAILIpiyE5TiUZvLd2c+D0KVHifJrmohPWwtFM4Cv74lMHPaTowfPHNM+ocx3BO7+aJSFE8yfAf5drAkw+4Qi91ZqzdhwYw8bIp83zKfpE0Jw0XOIuIG1fxOZbr/D9/kTgVG2y3WGUERYMxfBa+ukAb1AsCwz2R4n8R9ZAUwJKSYjyhiJP2uTkUJUJIR4vgwdgWiiit5LR80o2uRL672xDHTL0tvGYD1WQf5nJ7nD6OeDJb/25r/srH2+v/ZD9zM/wiMjOpm/IDZbCkX27a49u88HzVmmgv4+M2ttOb99QpHzx3khiw0zf5fJZv1kyuiP9R9D4OTL5rbulSNrRnYPMQk7ZTCV69uUUXSIDwnE8xFnE5OV52zAeQqfu9Z339o7P2WLzQyvTVvMnI3VddOWpdtwIGDJ5nN2KR3JX5YZRb6HpWys2ZNs3uuo+2PTGHetnet3CUXOWKIgYdn6OQMJ9taWoLG2o6zxWiiSfaUcn/sTXzeNbZ+fr/f/prPffbed5ev57PLDXULRX+9Z97VQdMPxu8xeUPZQ1qysJ7+7Z7bl7D/Fh/SRkkXIl2olUAIlUAIlUAIlUAIlUAIlUAIlUAIlUAJXCQHBL2+3C8LqcaAnyDLwSAQR2Bc8F/AVVFz2fJHpQFhQtk6gbjYG1yhepgYRhkhAVCEIEY/8TE8UbzwrZ0dwEGQV/CJELM01ApZKIn3dEAA8X7k3AWLXP2pR+o34QngQxBXkEvyULSUQetMkMmOIAt78NwaBXoEvwot+NjO7ZL28smEEPYk0s++S3k0CZoLobN5f9gqOeAj4YmN+gtgCcPqreItfqSTN5c1FAM7b/LtKuBFVsCM6LW0ba/M2X1kPMij8IQRYM8Fs89XvyNoaK0FA8FwfCsKBrIalEbgIC4K3TNaNzArrss2Uf8NDJpHydrIuZGMx8zAGAW1+IRvEOgkeH9d/BC+tK75ESG/J4+q/axP8JGQR05SawovvuAfBT48U682nlNAybj19CF8EQuNc2tqXzNfeEKR3vXv6t0CxNbFXdjFYj/ddxh553uijQgDjQ1MU2eX71m7ph8RJ4obv8kGBc35BIBKYN15BY+sja2zfnt21Xvaonl4yceyhpRFz514WoJ9ZJBh7Pn+4+VhT+87ZpAQjUVspOmsrA47wyI/wtVZrs45EAIKnMor8USD+/kM0Wl5PPNznQ84qAXl7igmq+6OE39J2re+uz6wFFvaXdV2eufz2wePc/vFxfhEp+OZ6b0+fdQa7Rtk8/u3FABlDjP87d26xGvuu/YG53wVvONaPn6wzA92OKIHlprWTceoec38Yh/VzH2eG85JoQcBWYtQaM1lk5qoE59KOssZTKPI7wt5XvvBJI3PNWI/ze23T75LlnjMXe5dtO8sP8UPCDm7zbLIn+IoxWwe/z/kHRpvWXS8igqLf584o9/M7iXjK/D8Bn7M3/K60V75lPNMZZB9VKFo5X/9ZAiVQAiVQAiVQAiVQAiVQAiVQAiVQApedwLJH0WWfy9U+foFGYs4uMelqZ9D5XU4CSkHKHJLxVyuBK01gCkWbyqOex9iu5FlOvCMKXfc8JtpnlEAJlEAJlEAJlEAJlEAJlEAJlEAJlEAJXA4CFYouxzoZJYFI5leD7Zdnza71kcoEkVGkV9gyU/Fa59L5X1kCm3oUneeIruRZvqlH0XnOvc8qgRIogRIogRIogRIogRIogRIogRIogRK4gAQqFF3ARVkMaZYTul2SWy/6CF3sUXd0JfD/Ebje6D2lvF2tBC4KgSshFF2Us7xC0UXxwo6jBEqgBEqgBEqgBEqgBEqgBEqgBEqgBC4QgQpFF2gxNgxF/6X3Gv1a9GSa/Wgu9qg7uhIogRK4uAT0a9LfSo8ifcr0bTpruwhn+Wsm+drRM8949Epa96I7aw69fwmUQAmUQAmUQAmUQAmUQAmUQAmUQAmUwAUkUKHoAi5Kh1QCJVACJVACJVACJVACJVACJVACJVACJVACJVACJVACJVAC50GgQtF5UO4zSqAESqAESqAESqAESqAESqAESqAESqAESqAESqAESqAESuACEvh/lPziGccWbPQAAAAASUVORK5CYII=
```

Comment:
The fixes to the sizing logic made this control image being incorrect visible...we _we're_ clipping the *bottom* off.